test/command/test-check-performance-regression.rb

```diff
@@ -0,0 +1,272 @@
+# coding: utf-8
+# Copyright (C) 2019  Kentaro Hayashi <hayashi@clear-code.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+require "groonga-query-log/command/check-performance-regression"
+
+class CheckPerformanceRegressionCommandTest < Test::Unit::TestCase
+  include Helper::Path
+
+  def setup
+    @command = GroongaQueryLog::Command::CheckPerformanceRegression.new
+  end
+
+  def fixture_path(*components)
+    super("check-performance-regression", *components)
+  end
+
+  sub_test_case("options") do
+
+    MISSING_QUERY_LOG_ERROR = <<-OUTPUT
+query logs is not specified. use --input-old-query and --input-new-query
+    OUTPUT
+
+    def run_command_with_stderr
+      actual = ""
+      Tempfile.open("redirect-stderr-tmpfile") do |file|
+        $stderr.reopen(file)
```

Comment:
We should revert to the original `$stderr`.

Reply:
I've fixed to recover.